sormas-app/app/src/main/java/de/symeda/sormas/app/backend/sample/PathogenTestDtoHelper.java

```diff
@@ -63,6 +69,12 @@ protected void fillInnerFromDto(PathogenTest target, PathogenTestDto source) {
         target.setTestType(source.getTestType());
         target.setTestedDisease(source.getTestedDisease());
         target.setTestedDiseaseDetails(source.getTestedDiseaseDetails());
+
+        //newly added
```

Comment:
Please only add comments that provide relevant information to understand the code or what it's supposed to do